src/lib/api.ts

```
@@ -38,452 +38,464 @@
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ customization }),
+      }
+    );
+
+    if (response.status == 400) {
+      throw Error("Error while running API");
+      return false;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)